docs/_includes/generated/security_configuration.html

```diff
@@ -10,7 +10,7 @@
         <tr>
             <td><h5>security.ssl.algorithms</h5></td>
             <td style="word-wrap: break-word;">"TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_DHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384"</td>
-            <td>The comma separated list of standard SSL algorithms to be supported. Read more &#60;a href="http://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#ciphersuites"&#62;here&#60;/a&#62;.</td>
+            <td>The comma separated list of standard SSL algorithms to be supported. Read more &lt;a href="http://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#ciphersuites"&gt;here&lt;/a&gt;.</td>
```

Comment:
this doesn't look correct.

Reply:
Why do you think so? &gt; and &lt; are valid html encodings(I think it is even more common way of escaping < and >). I've changed it, cause it was impossible to retrieve the other version from `Jsoup` which is used in tests. I've also checked it renders well both in Chrome as well as in Firefox.

Reply:
it shouldn't be escaped at all, no?

Reply:
since it's part of a link

Reply:
ah you didn't escape any descriptions yet, nvm.

Reply:
Aaa, that's what you meant. I haven't migrated the actual descriptions to use the new feature yet. Wanted to do it a separate commit/PR. First wanted to establish the API/feature.

Reply:
Should've added comment to the PR.